src/pages/ExamsPage/components/ResetExamAttemptModal.jsx

```diff
@@ -1,22 +1,34 @@
 import PropTypes from 'prop-types';
 
+import { useState } from 'react';
 import {
-  Button, useToggle, ModalDialog, ActionRow,
+  Button, useToggle, ModalDialog, ActionRow, StatefulButton,
 } from '@edx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import { useDeleteExamAttempt } from '../hooks';
 import messages from '../messages';
 
-const ResetExamAttemptButton = ({ username, examName, attemptId }) => {
+const ResetButtonProps = {
+  labels: {
```

Comment:
What happens to this button while it's in the `pending` state? Is it still clickable? We should probably add the `disabledStates` prop to this object to disable the button while it's in the `pending` state (and maybe `complete`, as well).

Reply:
The button isn't clickable in this state.